components/clm/bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -236,7 +236,7 @@ lnd/clm2/surfdata_map/surfdata_64x128_simyr2000_c170111.nc</fsurdat>
 <fsurdat hgrid="NLDAS"    sim_year="2000" irrig=".false." use_crop=".false." >
 lnd/clm2/surfdata_map/surfdata_nldas2_simyr2000_c181207.nc</fsurdat>
 
-<fsurdat hgrid="0.125x0.125"  sim_year="2000" use_crop=".false." >
+<fsurdat hgrid="r0125xr0125"  sim_year="2000" use_crop=".false." >
```

Comment:
Should this be `r0125` instead of `r0125xr0125`? In config_grids.xml, it is referred to as `r0125` everywhere, and the only time we have a `r0125xr0125` is in the grid alias for a coupled configuration when all components are on the same grid.